synapse/storage/schema/delta/53/account_validity.sql

```diff
@@ -0,0 +1,22 @@
+/* Copyright 2019 New Vector Ltd
```

Comment:
The schema has been bumped so this needs to go in `/54/`, and update `SCHEMA_VERSION` to be 54 in `synapse.storage.prepare_database`